cardano-testnet/src/Testnet/Commands/Governance.hs

```diff
@@ -0,0 +1,47 @@
+
+
+module Testnet.Commands.Governance
+  ( createByronUpdateProposal
+  , createByronUpdateProposalVote
+  ) where
+
+import           Prelude
+
+import           Control.Monad.Catch
+import           Control.Monad.IO.Class
+
+import           Testnet.Util.Process (execCli_)
+
+import           Hedgehog.Internal.Property
+
+
+createByronUpdateProposal
```

Comment:
For each of these functions, you will need `HasCallStack` and `GHC.withFrozenCallStack`.

This will ensure than when that command fails, the failure will be marked in the right location in the test (rather than in this file, which is not as useful).

See https://github.com/input-output-hk/cardano-node/blob/newhoggy/update-ouroboros-network/cardano-testnet/src/Testnet/Util/Process.hs#L63-L68 for examples

Reply:
Fixed!